charts/devtron-backups/values.yaml

```diff
@@ -33,7 +35,7 @@ global:
       # Only Encoded JSON CREDENTIALS allowed. Example shown below.
```

Comment:
Specify base64 encoded instead of only encoded

Reply:
Done!